tests/Secrets/tst_dataprotection/tst_dataprotection.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2017 Jolla Ltd.
+ * Contact: Chris Adams <chris.adams@jollamobile.com>
```

Comment:
Wrong contact and year

Reply:
We use the same header everywhere. Did not want to alter it just for this one file.

Reply:
But this still not Chris' rather yours.